api/app/schemas/hfi_calc.py

```diff
@@ -149,12 +139,12 @@ class DateRange(BaseModel):
     start_date: Optional[date]
     end_date: Optional[date]
 
-
-class HFILoadResultRequest(BaseModel):
-    """ Request to load the HFI Calculator. """
-    start_date: Optional[date]
-    end_date: Optional[date]
-    selected_fire_center_id: int
+    def num_days(self) -> Optional[int]:
```

Comment:
Tests?

Reply:
It's covered in higher level tests - you want a unit test specifically for that?

Reply:
I would like unit tests for the +1 on end date yes